beast/plotting/plot_triangle.py

```diff
@@ -3,9 +3,9 @@
 from astropy.io import fits
 import copy
 
+
 def plot_triangle(
-    beast_stats_file,
-    param_list=['Av', 'Rv', 'logA', 'f_A', 'M_ini', 'Z', 'distance'],
+    beast_stats_file, param_list=["Av", "Rv", "logA", "f_A", "M_ini", "Z", "distance"],
```

Comment:
I'm puzzled by black moving function inputs back into the same line.  I guess it's because the number of characters is still small enough?